bin/start_backend.sh

```diff
@@ -80,18 +83,26 @@ if [ $? -ne 0 ]; then
     exit 1
 fi
 
-# JEMALLOC enable DEBUG 
-# export JEMALLOC_CONF="junk:true,tcache:false,prof:true"
+# enable jemalloc
+JEMALLOC_LIB=$STARROCKS_HOME/lib/libjemalloc.so
+ln -s -f $STARROCKS_HOME/lib/libjemalloc.so.2 $JEMALLOC_LIB
```

Comment:
how about putting the debug and release .so into different folder, so no need to create the link during the start script, just set the LD_LIBRARY_PATH to different sub folder to take effect.

Reply:
need to consider running in container env where the fs in possibly readonly.

Reply:
Our BE writes pid files in the bin directory at startup, for symbolic links it is similar.